src/tracker/tracker-funcs.jl

```diff
@@ -507,13 +507,13 @@ Convert the centroid coordinates from row and column to latitude and longitude d
 """
 function convertcentroid!(propdf, latlondata, colstodrop)
     latitude, longitude = [
-        [latlondata[c][x, y] for
+        [latlondata[c][Integer(round(x)), Integer(round(y))] for
          (x, y) in zip(propdf.row_centroid, propdf.col_centroid)] for
         c in ["latitude", "longitude"]
     ]
 
     x, y = [
-        [latlondata[c][z] for z in V] for
+        [latlondata[c][Integer(round(z))] for z in V] for
```

Comment:
Using `Integer` for casting seems to work. However,  `Integer` [is an abstract type](https://docs.julialang.org/en/v1/base/numbers/#Core.Integer). For casting to concrete types, consider using `Int`.

Reply:
Good catch! It looks like `Integer(foo)` defaults to `Int64(foo)` on my machine. The docs [^1] say that `Base.Int` is the default type for array indices, and its concrete type depends on the system word size. Let's go for that.

[^1]: https://docs.julialang.org/en/v1/base/numbers/#:~:text=This%20is%20the%20default%20type%20of%20most%20integer%20literals%20and%20is%20an%20alias%20for%20either%20Int32%20or%20Int64%2C%20depending%20on%20Sys.WORD_SIZE.%20It%20is%20the%20type%20returned%20by%20functions%20such%20as%20length%2C%20and%20the%20standard%20type%20for%20indexing%20arrays.